.github/workflows/test_code.yml

```diff
@@ -10,10 +10,12 @@ jobs:
         uses: actions/setup-python@v1
         with:
           python-version: '3.6'
+      - name: install mypy
+        run: pip3 install mypy==0.770
+      - name: install requirements
+        run: pip3 install -r requirements.txt
```

Comment:
we don't need to install all dependencies (look above)

Reply:
Maybe not all, but I don't want to specify all used modules here one by one (otherwise this will have to be updated for every new typing in our dependencies). Is there any downside to installing them all? (other than execution time)

Reply:
No, just execution time.